assets/dev/js/editor/components/template-library/constants.js

```diff
@@ -5,3 +5,10 @@ export const SAVE_CONTEXTS = Object.freeze( {
 	BULK_MOVE: 'bulkMove',
 	BULK_COPY: 'bulkCopy',
 } );
+
+export const QUOTA_WARNINGS = Object.freeze( {
+	/* Translators: 1: Quota usage percentage */
+	warning: __( 'You’ve reached %1$d%% of your storage.', 'elementor' ),
```

Comment:
```suggestion
	warning: __( 'You\'ve reached %1$d%% of your storage.', 'elementor' ),
```
Keep it consistent with the one below.